gammapy/detect/cwt.py

```diff
@@ -73,25 +73,29 @@ class CWT(object):
     background model to which was added the flux of the extragalactic background,
     and the exposure for the considered energy range were used.
 
+    Read more in ???.
 
     TODO: instead of storing all the arrays as data members,
     we could have a ``.data`` member which is a dict of images?
 
     Parameters
     ----------
-    min_scale : float
-        first scale used
-    nscales : int
-        number of scales considered
-    scale_step : float
-        base scaling factor
+    n_scale : int
```

Comment:
Can I do parameters `n_scale`, 'min_scale' and 'step_scale' optional and if I can what default values I can setup?